docs/hack_loki_operator.md

```diff
@@ -0,0 +1,280 @@
+# Hacking on Loki Operator
+
+Loki Operator is the Kubernetes Operator for [Loki](https://grafana.com/docs/loki/latest/) provided by the Red Hat OpenShift engineering team.
```

Comment:
Please remove the reference `provided by the Red Hat OpenShift engineering team` here.